doc/src/modules/functions/elementary.rst

```diff
@@ -1,78 +1,130 @@
 Elementary
 ==========
 
-This module implements elementary functions, as well as functions like ``Abs``,
-``Max``, etc.
+This module implements elementary functions such as trigonometric, hyperbolic
+as well as functions like ``Abs``, ``Max``, ``sqrt`` etc.
```

Comment:
I meant to say that the sentence as it is erroneously indicates that `sqrt` is not an elementary function (but one of the "functions like").
So it should be `elementary functions such as trigonometric, hyperbolic, and sqrt, as well as functions like ...`.